crowbar_framework/app/models/service_object.rb

```diff
@@ -304,7 +311,7 @@ def restore_to_ready(nodes)
   # Receives proposal info (name, barclamp), list of nodes (elements), on which the proposal
   # should be applied, and list of dependencies - a list of {barclamp, name/inst} hashes.
   # It adds them to the queue, if possible.
-  def queue_proposal(inst, elements, deps, bc = @bc_name)
+  def queue_proposal(inst, element_order, elements, deps, bc = @bc_name)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for queue_proposal is too high. [58.35/50]